app/components/newsletter/newsletter-header-overview.js

```diff
@@ -181,6 +189,34 @@ export default class NewsletterHeaderOverviewComponent extends Component {
     this.showConfirmPublishAll = false;
   }
 
+  async canSendMailCampaign() {
+    const documentPublicationActivity = await this.args.meeting.internalDocumentPublicationActivity;
+
+    let hasDocumentPublicationDate = isPresent(documentPublicationActivity.plannedDate);
+    let hasNotas = false;
+    let hasThemes = false;
+
+    const agendaitems = await this.args.agenda.agendaitems;
+    for (const agendaitem of agendaitems.toArray()) {
+      const agendaitemTreatment = await agendaitem.treatment;
+      const newsletterItem = await agendaitemTreatment.newsletterInfo;
+      const themes = await newsletterItem?.themes;
+      if (themes?.length) {
+        hasThemes = true;
+      } else {
+        hasThemes = false;
+        break
+      }
+
+      const type = await agendaitem.type;
+      if (type.uri === CONSTANTS.AGENDA_ITEM_TYPES.NOTA) {
+        hasNotas = true;
```

Comment:
when there is 1 nota, you can break the loop.
But since the loop also checks the themes, you should break the for loop if both `hasThemes` and `hasNotas` are true at any stage.

Reply:
check the other comments too, if we query we could already filter on notas.